### PR TITLE
changed: pass table ID into SimpleTable constructors

### PR DESCRIPTION
**Pull request by akva2**
use this to improve logging in case of failure

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Review by blattms — APPROVED** [review]
Looks good to me. Anything that helps with error messages is highly appreciated.

File: opm/input/eclipse/EclipseState/Tables/TableSchema.hpp [hunk 1]
```diff
@@ -30,6 +30,13 @@ namespace Opm {
 
     class TableSchema {
     public:
+        TableSchema() = default;
+
+        TableSchema(const OrderedMap<std::string,ColumnSchema>& columns)
+          : m_columns(columns)
+        {
+        }
```
Comment (bska) [inline opm/input/eclipse/EclipseState/Tables/TableSchema.hpp:38]: Just out of curiosity, why do we need these constructors?

Comment (akva2) [reply opm/input/eclipse/EclipseState/Tables/TableSchema.hpp:38]: woops, we don't, not supposed to be in here.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks.  This is greatly appreciated.  Please merge when green.

**Pull request merged**